SU-HSDO/suhumsci
# Shs 22 megamenu v1 wcag

## Summary
Fixes megamenu V1 keyboard tab accessibility issue.

## Steps to Test
1. Ensure that the old megamenu is being used
2. Using the keyboard, tab through the main navigation and ensure that:
3. The outline color is now using the 'primary' outline color and
4. Any menu item that is a link and a button (Training for me on Traditional) that the button now receives an outline
5. Bonus step: verify it works with VoiceOver. I did, and it's fine.

## Screenshots
![Screen Shot 2022-09-20 at 3 27 54 PM](https://user-images.githubusercontent.com/8405274/191347884-f1bfbcea-9555-4bd2-925f-8511a6aa7c08.png)

## Notes
- When going over the accessibility for the menu, I noticed a lot of other things in general on the homepage. They are included in the ticket here: https://fourkitchens.atlassian.net/browse/SHS-22
- This removes the `outline: none;` from the button mixin. This is global. Looking over a few pages, I don't see any adverse affect, but this shouldn't have been included in the first place.
